src/app/progress-bar/progress-bar.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-progress-bar',
+  templateUrl: './progress-bar.component.html',
+  styleUrls: ['./progress-bar.component.scss']
+})
+export class ProgressBarComponent implements OnInit {
+  _progressValue: number | null = 0;
```

Comment:
Is it really nullable?

Reply:
This was to prevent TS error from strict mode -> changed to non-null assertion operator in sold-products template